index.html

```diff
@@ -2874,59 +2909,61 @@ <h2>Processing a Manifest</h2>
 								object in which one or more properties expect entities.</p>
 						</li>
 
-						<li>
+						<li id="processing-checks-language">
 							<p>(<a href="#manifest-lang-dir-global"></a> and <a href="#manifest-lang-dir-local"></a>)
-								Recursively check that every <var>language</var> property in <var>processed</var> is
-								well-formed [[!bcp47]] and issue a warning for any invalid values found.</p>
+								Recursively check that every <var>language</var> property in <var>processed</var> is <a
+									href="https://tools.ietf.org/html/bcp47#section-2.2.9">valid</a>&#160;[[!bcp47]] and
+								issue a warning for any invalid values found.</p>
 						</li>
 
```

Comment:
I think we should keep to the 'well-formed' for now (that is used in other parts of the spec). The response to the 'valid' question should come from the I18N guys, and, afaik, @r12a will raise an issue about it in an I18N repository. (We may want to add an editorial note here, just that we do not forget about it...)

Reply:
Oops, I thought I undid them all! I was intending to leave it alone pending a resolution. I'll run another search on valid and add a note.